connector/connect/server/src/main/scala/org/apache/spark/sql/connect/ui/SparkConnectServerHistoryServerPlugin.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.connect.ui
+
+import org.apache.spark.SparkConf
+import org.apache.spark.scheduler.SparkListener
+import org.apache.spark.status.{AppHistoryServerPlugin, ElementTrackingStore}
+import org.apache.spark.ui.SparkUI
+
+class SparkConnectServerHistoryServerPlugin extends AppHistoryServerPlugin {
+
+  override def createListeners(
+      conf: SparkConf,
+      store: ElementTrackingStore): Seq[SparkListener] = {
+    Seq(new SparkConnectServerListener(store, conf))
+  }
+
+  override def setupUI(ui: SparkUI): Unit = {
+    val store = new SparkConnectServerAppStatusStore(ui.store.store)
+    if (store.getSessionCount > 0) {
+      new SparkConnectServerTab(store, ui)
+    }
+  }
+
+  override def displayOrder: Int = 1
```

Comment:
increase the display order to avoid conflict with StreamingQueryHistoryServerPlugin?

Reply:
It'd be good to have this, but because of how other SHS plugins are also broken in the same way, it may be clearer to have a separate PR that fixes the displayOrder of all of the plugins in one go so that we can see the context of why the values were chosen in one single PR.

e.g. if we set this plugin's display order to `4`, it wouldn't be very obvious just from this PR alone why it should be a `4` while a lot of other plugins have a display order of `1`

Reply:
Agreed we should update them all at once. Should we do it in a separate PR or just update all of them in this one?

Reply:
I prefer doing the wholesale update in a separate PR, but please check with the Apache Spark committers in this thread to confirm 🙏

Reply:
Hm, I checked and since this is only incorrect in one other place (StreamingQueryHistoryServerPlugin.scala), I just updated it in this PR